### PR TITLE
Related articles

### PR DESCRIPTION
**Pull request by georgecrawford**
https://ft-google-amp-staging-pr-23.herokuapp.com/content/94e97eee-ce9a-11e5-831d-09f7778e7377

screenshot 2016-02-19 10 20 11

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require('isomorphic-fetch');
```
Comment (matthew-andrews) [inline app.js]: suggest `GLOBAL.fetch = require('node-fetch');` unless you need server side

Comment (georgecrawford) [inline app.js]: Thanks Matt

**Comment by georgecrawford** [comment]
@quarterto - I wonder if we need to filter the list of related articles to check if they're accessible to the current user?

File: views/article.html [hunk 2]
```diff
@@ -53,7 +53,24 @@ <h1 itemprop="headline">{{title}}</h1>
...
+
+			{{#if relatedContent}}
+			<aside class="related-content">
+				<h2 class="related-content__header">Related articles</h2>
+				<ul class="related-content__items">
+					{{#each relatedContent}}
+ 					<li class="related-content__item">
+ 						<div class="related-content__container">
+							{{{date}}}
+							<a class="related-content__link" href="/content/{{{id}}}">{{{title}}}</a>
```
Comment (quarterto) [inline views/article.html]: These should link to ft.com. Relative links won't work as the page won't be viewed from amp.ft.com

File: scss/style.scss [hunk 3]
```diff
@@ -177,3 +177,45 @@ body {
...
+	padding: 0;
+	@include oGridRow();
+}
+
+.related-content__item {
+	@include oGridColspan((default: 12, M: 6));
+	margin-bottom: oGridGutter(L);
+
+	@include oGridRespondTo(M) {
+		display: -webkit-flex;
```
Comment (georgecrawford) [inline scss/style.scss]: @quarterto I could add autoprefixer to the build step if you'd prefer?

**Comment by jose-corral-ft** [comment]
Issues: 
- small screen devices: table needs margin on the right side. 
img_0030-2

- iPad iOS 8.*: Components are present on a single row (on iOS 9 it does a table of 2xN rows).  

img_0130

**Comment by jose-corral-ft** [comment]
Small screen issue addressed.

**Comment by georgecrawford** [comment]
Don't merge this yet please - I'm going to address Gadi's advice re. data sources.

**Pull request merged**